Final_project/sql/transfer_customers_from_raw_to_bronze.sql

```diff
@@ -0,0 +1,21 @@
+DELETE FROM `{{ params.project_id }}.1_bronze.customers`
+WHERE DATE(_logical_dt) = "{{ ds }}";
+
+MERGE   `{{ params.project_id}}.1_bronze.customers` T
```

Comment:
I would transfer the data to Bronze layer "as is" and do the merge in Silver. Bronze should have 1-to-1 correspondence with the raw data in the bucket. This is the best practice.

Reply:
Also, if you DELETE the partition, there's no sense in MERGE. You should use one of them.